src/main/java/tobii/Tobii.java

```diff
@@ -14,15 +14,19 @@
 
     private static String OS = System.getProperty("os.name").toLowerCase();
 
+    public static float[] getGazePosition;
+
     public static float[] gazePosition() {
```

Comment:
I suppose it would be a breaking change, but wouldn't it make more sense for the method to be `getGazePosition` and the variable to be `gazePosition`?

Reply:
Yes, my bad, I will fix that.